cmd/tokengen/main_test.go

```diff
@@ -53,7 +53,7 @@ func TestGen(t *testing.T) {
 					driver,
 					"--auditors", "aOrg1MSP,b:Org2MSP",
 				},
-				ErrMsg: "Error: failed to generate public parameters: failed to get auditor identity [aOrg1MSP]: invalid input [aOrg1MSP]",
+				ErrMsg: "Error: failed to generate public parameters: failed to get auditor identity [aOrg1MSP]: failed to create x509 provider for [aOrg1MSP]: could not load a valid signer certificate from directory aOrg1MSP/signcerts: stat aOrg1MSP/signcerts: no such file or directory",
```

Comment:
Do you want to check for the "entire" error msg? Would it be enough to work with `contains`? If we change the error message returned by some function that extends this message in some form, we always need to update this test.

Reply:
it is just to make sure of the reason why it fails